drafts/25-xxx-specifications.txt

```diff
@@ -704,125 +747,133 @@ have consensus amongst the authors:
       below.
 
-ex17. Preprocessing computes the integer value of conditional expressions
-      using the greatest integer range available to the processor to
-      determine the truth or falsity of the controlling expression.
+ex17. Preprocessing computes the integer value of conditional
+      expressions using the greatest integer range available to the
+      processor to determine the truth or falsity of the controlling
+      expression.
 
-ex20. The processor shall reject a program if evaluation of
-      the expression generates a computational error (such 
-      as divide by zero).
+ex20. The processor shall reject a program if evaluation of the
+      expression generates a computational error (such as divide by
+      zero).
 
-ex25. When the expression evaluates to zero, the controlling expression
-      will be considered "false". If the expression evaluates to
-      any non-zero value, the controlling expression will be considered
-      "true".
+ex25. When the expression evaluates to zero, the controlling
+      expression will be considered "false". If the expression
+      evaluates to any non-zero value, the controlling expression will
+      be considered "true".
 
 
 5.1 Operators allowed in controlling expressions
 ------------------------------------------------
 
-op01. To maintain compatibility with the use of C preprocessing directives
-      in many existing Fortran programs, the operators allowed in
-      controlling expressions in #if and #elif expressions are a subset of
-      those defined in the C 2023 standard §6.5 "Expressions" and §6.6
-      "Constant expressions".
+op01. To maintain compatibility with the use of C preprocessing
+      directives in many existing Fortran programs, the operators
+      allowed in controlling expressions in #if and #elif expressions
+      are a subset of those defined in C 2023 section 6.5
+      "Expressions" and section 6.6 "Constant expressions".
 
-op03. A "precedence level" is assigned to each operator that determines
-      how the operators combine with sub-expressions containing other operators
-      at different precedence levels.
+op03. A "precedence level" is assigned to each operator that
+      determines how the operators combine with sub-expressions
+      containing other operators at different precedence levels.
 
-op05. An "associativity" is assigned to each operator that determines how
-      operators at the same precedence level are combined.
-          - "left" means the operator binds to the left,
-          - "right" means the operator binds to the right.
-          - "nonassoc" means that the operator is not associative.
+op05. An "associativity" is assigned to each operator that determines
+      how operators at the same precedence level are combined. -
+      "left" means the operator binds to the left, - "right" means the
+      operator binds to the right. - "nonassoc" means that the
+      operator is not associative.
```

Comment:
This bulleted list has been converted to a paragraph, but the bullets were not removed.

Also, "nonassoc" no longer appears in the table, so we can remove that.

```suggestion
op05. An "associativity" is assigned to each operator that determines
      how operators at the same precedence level are combined. "left"
      means the operator binds to the left, "right" means the operator
      binds to the right.
```